### PR TITLE
updating logger to latest

### PR DESCRIPTION
**Pull request by paganotoni**

### CODE REVIEW

File: param_logger.go [hunk 1]
```diff
@@ -1,50 +1,97 @@
...
+	"PasswordConfirmation",
+	"CreditCard",
+	"CVC",
+}
+
+var filteredIndicator = []string{"[FILTERED]"}
+
+// ParameterLogger logs form and parameter values to the logger
+type parameterLogger struct {
+	blacklist []string
```
Comment (mclark4386) [inline param_logger.go]: Just to point it out here as well, so it's not forgotten, @markbates want this changed to `exclusionlist` as well.

File: param_logger.go [hunk 2]
```diff
@@ -1,50 +1,97 @@
...
+}
+
+// ParameterLogger logs form and parameter values to the loggers
+func ParameterLogger(next buffalo.Handler) buffalo.Handler {
+	pl := parameterLogger{
+		blacklist: ParameterExclusionList,
```
Comment (mclark4386) [inline param_logger.go]: Also here.

File: param_logger.go [hunk 3]
```diff
@@ -56,10 +103,44 @@ func multipartParamLogger(mp *multipart.Form, c buffalo.Context) error {
...
 	c.LogField("form", string(b))
 	return nil
 }
+
+//maskSecrets matches ParameterExclusionList against parameters passed in the
+//request, and returns a copy of the request parameters replacing blacklisted params
+//with [FILTERED].
+func (pl parameterLogger) maskSecrets(form url.Values) url.Values {
+	if len(pl.blacklist) == 0 {
+		pl.blacklist = ParameterExclusionList
```
Comment (mclark4386) [inline param_logger.go]: Which means these and below will need to change as well ... probably best to do a find and replace in this case^^

File: param_logger_test.go [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+	r.Equal(filteredForm.Get("pAssWorD"), filteredIndicator[0])
+	r.Equal(filteredForm.Get("PasswordConfirmation"), filteredIndicator[0])
+	r.Equal(filteredForm.Get("LastName"), "Pagano")
+	r.Equal(filteredForm.Get("SomeCVC"), "Untouched")
+}
+
+func Test_maskSecretsCustom(t *testing.T) {
+	r := require.New(t)
+	pl := parameterLogger{
+		blacklist: []string{
```
Comment (mclark4386) [inline param_logger_test.go]: Don't forget here^^

**Review by mclark4386 — CHANGES_REQUESTED** [review]
Just a reminder of the changes requested in the other repo^^

**Comment by stanislas-m** [comment]
@paganotoni I've changed things in your PR, can you confirm it still works the way you want?

I've tried to refactor it so it's closer to the original, and since all the config is handled by `ParameterExclusionList`, there is no need to create a whole structure for that. Thoughts?

**Comment by stanislas-m** [comment]
I've reverted my changes, and updated from latest upstream version.

**Pull request merged**